test/provision/k8s_install.sh

```diff
@@ -162,8 +182,15 @@ sudo iptables --policy FORWARD ACCEPT
 
 #check hostname to know if is kubernetes or runtime test
 if [[ "${HOST}" == "k8s1" ]]; then
+    case $K8S_VERSION in
```

Comment:
Due that we already have versions for this, and to avoid this if, why we don't have a $KUBEADM_CONFIG, and by default is `KUBEADM_CONFIG=$KUBEADM_CONFIG_ALPHA1` and in the 1.12 option you change to $KUBEADM_CONFIG as `$KUBEADM_CONFIG_ALPHA2`

Like the DNS deployment.

Reply:
Done